src/main/resources/l10n/Menu_el.properties

```diff
@@ -1,31 +1,30 @@
+#!
```

Comment:
It seems that something went wrong when merging the language file here? Why is there a file header? Also, why are some of the localizations removed?